lib/iphone-simulator-xcode-7.ts

```diff
@@ -57,7 +57,7 @@ export class XCode7Simulator extends IPhoneSimulatorNameGetter implements ISimul
 			let launchResult = this.simctl.launch(device.id, applicationIdentifier).wait();
 
 			if (options.logging) {
-				this.printDeviceLog(device.id, launchResult);
+				this.getDeviceLogProcess(device.id, launchResult);
```

Comment:
You can consider returning this value here so that a consumer of the ios-sim-portable API wouldn't be required to call `getDeviceLogProcess` after calling `run`. Just a thought/suggestion and definitely not a merge-stopper though.